kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/orders/ordering/Order.java

```diff
@@ -0,0 +1,76 @@
+package com.kodilla.good.patterns.orders.ordering;
+
+import com.kodilla.good.patterns.orders.data.*;
+import java.time.LocalDate;
+
+public final class Order implements OrderData {
```

Comment:
As I mentioned previously OrderData is not necessary, and it is visible also in this example. Order implements OrderData and contains OrderData.